release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-22"]
+=== RHBA-2021:0825 - {product-title} 4.6.22 bug fix update
+
+Issued: 2020-03-23
+
+{product-title} release 4.6.22 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:0825[RHBA-2021:0825] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:0826[RHBA-2021:0826] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5894581[{product-title} 4.6.22 container image list]
+
+[id="ocp-4-6-22-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Zeroconf library did not correctly rate-limit multicast DNS (mDNS) responses. As a result, excessive mDNS traffic flooded network records. This update adds rate-limiting to the Zeroconf library, which significantly reduces mDNS traffic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1936539[*BZ#1936539*])
++
+* Previously, an API server could fail to create a resource, which would return a 409 status code when there was a conflict updating a resource quota. Consequently, the resource would fail to create, and you might have had to retry the API request. With this update, the OpenShift Console web application attempts to retry the request 3 times when receiving a 409 status code, which is often sufficient for completing the request. In the event that a 409 status code continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928230[*BZ#1938230*])
++
+* Previously, the livenessProbe would fail when the Prometheus container was under a heavy load (for example, during WAL replay). This heavy load would cause several issues and endless restart loops. This update removes livenessProbe, and as a result, heavy loads no longer cause endless restart loops. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1935586[*BZ#1935586*])
```

Comment:
- s/livenessProbe/liveness probe
- We typically try to stay away from parentheses. Maybe `...heavy load; for example, during WAL replay.`
- s/WAL/write-ahead logging (WAL)